src/commands/lambda/flare.ts

```diff
@@ -141,6 +142,30 @@ export class LambdaFlareCommand extends Command {
     const configStr = util.inspect(config, false, undefined, true)
     this.context.stdout.write(`\n${configStr}\n`)
 
+    // Get tags
+    this.context.stdout.write('\n🏷 Getting Resource Tags...\n')
+    const arn = config.FunctionArn ?? this.functionName
```

Comment:
Do we want to add this `?? this.functionName` to handle the case where `config.FunctionArn` is undefined?
Or would it be better practice to immediately write an error and exit when `config.FunctionArn` is undefined?

Reply:
FunctionArn will never be undefined, so we could do `config.FunctionArn!`.

I think it's a safe assumption. As per the things you said.